service/src/main/java/bio/terra/workspace/db/WorkspaceDao.java

```diff
@@ -64,13 +65,17 @@ public class WorkspaceDao {
                       .map(DbSerDes::jsonToProperties)
                       .orElse(null))
               .workspaceStage(WorkspaceStage.valueOf(rs.getString("workspace_stage")))
+              .workspaceLineage(
+                  Optional.ofNullable(rs.getString("workspace_lineage"))
+                      .map(
+                          lineage -> DbSerDes.fromJson(lineage, new TypeReference<List<UUID>>() {}))
+                      .orElse(null))
```

Comment:
Should this fill in an empty array instead of null?

Reply:
yeah.. i can. i've handled the null in the workspace class so it's not strictly necessary. 

i still think we need better static checker for null. We have very inconsistent nullable annotation and null check.